uECC.c

```diff
@@ -1009,8 +1016,10 @@ int uECC_shared_secret(const uint8_t *public_key,
                        const uint8_t *private_key,
                        uint8_t *secret,
                        uECC_Curve curve) {
-    uECC_word_t public[uECC_MAX_WORDS * 2];
+     
```

Comment:
yep sorry, I've already done that locally. Haven't pushed yet, apologies for that.